server/website/website/views.py

```diff
@@ -1233,7 +1239,7 @@ def give_result(request, upload_code):  # pylint: disable=unused-argument
 
     elif group_res.ready():
         assert group_res.successful()
-        next_config = JSONUtil.loads(next_config)
+        next_config = JSONUtil.loads(latest_result.next_configuration)
```

Comment:
Fix bug: we need to get the fresh next_configuration, not a cached one. Otherwise, it is possible that the celery status is `ready` but the next_config is `None`